spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -146,59 +146,91 @@ NOTE: Jakarta Data allows applications to intermix both patterns by defining met
 
 == Entity Classes
 
-In Jakarta Data, an entity refers to a fundamental data representation and management building block. It can be conceptually understood in several aspects:
+The notion of an _entity_ is the fundamental building block with which a data model may be constructed.
+Abstractly, an entity (or _entity type_) is a schema for data.
 
-1. *Entity Classes*: Entity classes are simple Java objects equipped with fields or accessor methods that designate each property of the entity. Depending on your data storage needs, you may use annotations from the Jakarta Persistence specification, such as `jakarta.persistence.Entity`, `jakarta.persistence.Id`, and `jakarta.persistence.Column`, to define and customize entities for relational databases. Alternatively, for NoSQL databases, you can use annotations from the Jakarta NoSQL specification, including `jakarta.nosql.Entity`, `jakarta.nosql.Id`, and `jakarta.nosql.Column`.
+- The schema may be as simple as a tuple of types, as is typical in the relational model, or it might be structured, as in document data stores.
+- The schema might be explicit, as in the case of SQL DDL declaring a relational table, or it might be implicit, as is commonplace in key/value stores.
+- Either way, we assume that the entity is represented in Java as a class, which we call the _entity class_. footnote:[We will not consider generic programs which work with entity data via detyped representations.]
 
-2. *Data Schema*: Abstractly, an entity or entity type serves as a schema for data. It defines the structure and properties of the data it represents. This schema can be as simple as a set of typed fields, similar to the relational model, or more structured, as found in document data stores. The schema can be explicitly defined, as in the case of SQL Data Definition Language (DDL) declarations for relational tables, or it can be implicit, common in key/value stores.
+NOTE: When there's no risk of confusion, we often use the word “entity” to mean the entity class, or even an instance of the entity class.
 
-3. *Persistence and Representation*: Entities are associated with persistent data, meaning the data outlives any specific Java process utilizing it. Each persistent instantiation of the schema is distinguishable by a unique identifier. For example, a row of a relational database table is identifiable by the value of its primary key. In Java, these entities are represented as classes, referred to as entity classes. It's important to note that multiple instances of the entity class within a Java program can represent a single persistent instance of the schema.
+Data represented by an entity is persistent, that is, the data itself outlives any Java process which makes use of it. Thus, it's necessary to maintain an association between instances of Java entity classes, and state held in a data store.
```

Comment:
Minor grammar correction to remove a comma,

```suggestion
Data represented by an entity is persistent, that is, the data itself outlives any Java process which makes use of it. Thus, it is necessary to maintain an association between instances of Java entity classes and state held in a data store.
```

Reply:
Thanks, good catch.